lib/que/middleware/worker_collector.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module Que
+  module Middleware
+    class WorkerCollector
```

Comment:
This is a lot of code for what could be one line in `bin/que` - is the idea that this would be reused by Que UI?